app/src/test/java/ingsw/group1/findmyphone/contacts/SMSContactConverterTest.java

```diff
@@ -0,0 +1,59 @@
+package ingsw.group1.findmyphone.contacts;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Test for {@link SMSContactConverter}
+ *
+ * @author Giorgia Bortoletti
+ */
+public class SMSContactConverterTest {
+
+    private static final String EX_VALID_ADDRESS = "+393478989890"; //for contact and peer
+    private static final String CONTACT_VALID_NAME = "NewContact";
+    private static final String CONTACT_EMPTY_NAME = "";
+
+    private SMSContact contactWithoutName, contactWithName; //a contact may or may not have a name
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)